frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```
@@ -114,10 +114,10 @@ action.GitMachete.BaseResetToRemoteAction.info-dialog.title=Git Machete Reset In
 action.GitMachete.BaseResetToRemoteAction.info-dialog.message.HTML=\
   <html>\
     Git Machete Reset will perform a reset of local branch <b>{0}</b> \
-    so that it will point to the same commit as its remote branch <b>{1}</b>. \
+    so that it will point to the same commit as its remote branch <b>{1}</b>.<br><br> \
     Once you perform the action, you will still be able to restore the pre-reset commit. \
-    To achieve that check out <b>{0}</b> and use \
-    <tt>git reset --keep {2}</tt>. \
+    To achieve that check out <b>{0}</b> and use<br><br> \
+    <tt>git reset --keep {2}</tt>.<br><br> \
```

```suggestion
    <tt>git reset --keep {2}</tt><br><br> \
```

As the period at the end is misleading ;p